CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ from the water zone if the volumes from the full grid geometry are included, and
 
 ### Added
 - [#662](https://github.com/equinor/webviz-subsurface/pull/662) - Added support in `WellCompletion` for connection history from summary data.
+- [#708](https://github.com/equinor/webviz-subsurface/pull/708) - Added support for new report format for `DiskUsage`, which improves the estimate of free disk space.
```

Comment:
Need to move this now since we waited with merge.

Reply:
Hmm, it is still under UNRELEASED, but seems like we have 2 `Added` sections (Added - Changed - Fixed - Added). 
Do we have a standard for the order of `Added`, `Changed` and `Fixed`? The natural is maybe that order (alphabetical), but e.g. for the `0.2.3` and `0.2.1` releases it was `Changed`, `Fixed`, `Added`.

Reply:
sorry, latest release is `0.2.4 `now 😅 and hadn't rebased......